configuration/analyzer_config.json

```diff
@@ -258,6 +258,39 @@
     "secrets": {},
     "params": {}
   },
+  "CapeSandbox" : {
+    "type" : "file",
+    "python_module" : "cape_sandbox.CAPEsandbox",
+    "description" : "Automatic scan of suspicious files using CapeSandbox API",
+    "disabled" : false,
+    "external_service" : true,
+    "leaks_info" : true,
+    "docker_based" : false,
```

Comment:
the default is `false` so better to remove this from here.

Reply:
Done this for `docker_based`